include/iocore/eventsystem/UnixSocket.h

```diff
@@ -49,24 +49,27 @@
 
 bool transient_error();
 
-class UnixSocket
-{
-public:
+struct UnixSocket {
+  int fd{NO_SOCK};
```

Comment:
I think this member should stay private.  If it needs to be readable, you can use an accessor.  I don't think we want anyone outside this class to write this value, right?

Reply:
There aren't supposed to be any invariants here... there actually is one that I didn't think of, because `is_ok()` doesn't check for all invalid values. Having to go through a getter is just an inconvenience if there is no gain from encapsulation.

Reply:
Keeping this private will make it easier to switch to RAII in the future. Done.